posts/2022-10-04-node-cli-api.markdown

```diff
@@ -0,0 +1,38 @@
+# Node-Api-Cli Update 2022-10-04 - 2022-10-18
```

Comment:
BTW, you might want to add a high-level summary section for marcomms, and a high-level status report section. This will reduce the level of noise introduced by stakeholders trying to translate our updates. Check the consensus updates to see if they make sense for you.

Reply:
Merged before I saw this. I'll have a look at the consensus updates

Reply:
We've also been updating the readme here and there as we learn what seems to work.